lib/vanagon/engine/pooler.rb

```diff
@@ -56,7 +56,7 @@ def read_vanagon_token(path = "~/.vanagon-token")
         absolute_path = File.expand_path(path)
         return nil unless File.exist?(absolute_path)
 
-        puts "Reading vmpooler token from: #{path}"
+        $stderr.puts "Reading vmpooler token from: #{path}"
```

Comment:
This is exactly the message that is screwing us up in CI by being printed to stdout.